graphing.py

```diff
@@ -56,12 +62,62 @@ def get_score_graph(self, type, base_path, file_name, card_info) -> str:
         )
         ax.add_artist(at)
         
-        plt.plot(episode_plots, score_plots)
-        plt.title(f"{type} score over episodes")
+        plt.plot(episodes, scores)
+        plt.title(f"{graph_type} score over episodes")
         plt.xlabel("Episode")
         plt.ylabel("Score")
-        file_name = f"{base_path}/{type}_score_graph_{file_name}.png"
+        file_name = f"{base_path}/{graph_type}_score_graph_{file_name}.png"
         plt.savefig(file_name)
         
         return file_name
-    
\ No newline at end of file
+
+    def save_stats(self, graph_type: GraphType, stats_for: dict):
+        from json import dump
+
+        chunk_size = floor(len(self.episodes) / self.NUMBER_OF_CHUNKS)
+
+        episodes, scores = self.__reduce(graph_type, chunk_size)
+        data = {
+            "label": ', '.join([str(x) for x in flatten_dict(stats_for).values()]),
```

Comment:
you dont need to flatten the dict, just extract the values (and change the info back :imp: )